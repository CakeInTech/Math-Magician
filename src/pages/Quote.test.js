import React from 'react';
import renderer from 'react-test-renderer';
import Quote from './Quotes';

it('renders correctly', () => {
  const Linker = renderer.create(<Quote />).toJSON();
  expect(Linker).toMatchSnapshot();
});
