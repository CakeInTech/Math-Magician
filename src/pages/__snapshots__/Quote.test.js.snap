// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="quotes_container"
  data-testid="quote"
>
  <p>
    " Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -William Paul Thurston"
  </p>
</div>
`;
